doc/main/tbb_userguide/Advanced_Topic_Other_Kinds_of_Iteration_Spaces.rst

```diff
@@ -72,4 +72,39 @@ along its longest axis. When used with ``parallel_for``, it causes the
 loop to be "recursively blocked" in a way that improves cache usage.
 This nice cache behavior means that using ``parallel_for`` over a
 ``blocked_range2d<T>`` can make a loop run faster than the sequential
-equivalent, even on a single processor.
+equivalent, even on a single processor. 
+
+Also, ``blocked_range2d`` allows to use different value types across
```

Comment:
The ``blocked_range2d`` allows you to use different value types for its two dimensions: *rows* (the first dimension) and *columns* (the second dimension).